cmd/kubelet/app/options/options.go

```diff
@@ -29,6 +30,26 @@ import (
 	"github.com/spf13/pflag"
 )
 
+const (
+	defaultRootDir             = "/var/lib/kubelet"
+	experimentalFlannelOverlay = false
+
+	// When these values are updated, also update test/e2e/framework/util.go
+	defaultPodInfraContainerImageName    = "gcr.io/google_containers/pause"
+	defaultPodInfraContainerImageVersion = "3.0"
+	// Auto detect cloud provider.
+	AutoDetectCloudProvider         = "auto-detect"
+	DefaultKubeletPodsDirName       = "pods"
+	DefaultKubeletVolumesDirName    = "volumes"
+	DefaultKubeletPluginsDirName    = "plugins"
+	DefaultKubeletContainersDirName = "containers"
```

Comment:
@saad-ali @thockin could you confirm whether it is possible, in the future, for a user to configure those values like "plugin" "volume" in the mounting paths? It seems for me that those names are used internally and users should not configure them.